base/setup/usetup/usetup.c

```diff
@@ -3060,7 +3060,7 @@ InstallDirectoryPage(PINPUT_RECORD Ir)
         Status = InitDestinationPaths(&USetupData, InstallDir, InstallPartition);
         if (!NT_SUCCESS(Status))
         {
-            DPRINT1("InitDestinationPaths() failed. Status code: 0x%lx", Status);
+            DPRINT1("InitDestinationPaths() failed. Status code: 0x%lx\n", Status);
```

Comment:
```suggestion
            DPRINT1("InitDestinationPaths() failed. Status 0x%lx\n", Status);
```
for shortness and consistency to the line above in partlist.c

Reply:
Removed.

Reply:
ok